components/transparent_rectangle.py

```diff
@@ -0,0 +1,23 @@
+from components import Component
+from canvas import Canvas, Colour, Coord
+from data import Data
+
+
+class TransparentRectangle(Component):
+    """A transparent rectangle behind the data overlay for data values to stand out
+    from a white background (white racetrack, lines, etc).
+    """
+
+    COLOUR = Colour.transparentBlack
+
+    def __init__(self, top_left_coords: Coord, bottom_right_coords: Coord):
+        self.top_left_coords = top_left_coords
+        self.bottom_right_coords = bottom_right_coords
+
+    def draw_base(self, canvas: Canvas):
+        canvas.draw_rect(
+            self.top_left_coords, self.bottom_right_coords, TransparentRectangle.COLOUR
```

Comment:
I thought we upped the line character limit to 88?

Reply:
We did in BOOST but never did here. Will have to get around to this sometime. (`black` should be reading the config from `pyproject.toml` though so it should still work fine in this repo)